k9mail/src/main/java/com/fsck/k9/view/RecipientSelectView.java

```diff
@@ -196,6 +197,22 @@ protected void onDetachedFromWindow() {
     }
 
     @Override
+    public void onFocusChanged(boolean hasFocus, int direction, Rect previous) {
+        super.onFocusChanged(hasFocus, direction, previous);
+        if (hasFocus) {
+            displayKeyboard();
+        }
+    }
+
+    private void displayKeyboard() {
+        InputMethodManager imm = (InputMethodManager) getContext().getSystemService(Context.INPUT_METHOD_SERVICE);
+        if (imm == null) {
+            return;
+        }
+        imm.showSoftInput(this, InputMethodManager.SHOW_IMPLICIT);
```

Comment:
Well, the method is called `focusChange()`, showing the keyboard as a result of that seems like an implicit thing. I'm not sure if where this even makes a difference, tbh, seemed to me like `SHOW_IMPLICIT` was used in every displayKeyboard snippet I've seen around. *shrug